mev_inspect/models/liquidations.py

```diff
@@ -12,6 +12,7 @@ class LiquidationModel(Base):
     debt_token_address = Column(String, nullable=False)
     debt_purchase_amount = Column(Numeric, nullable=False)
     received_amount = Column(Numeric, nullable=False)
+    received_token_address = Column(String, nullable=False)
```

Comment:
is it intentional that this is nullable=False but the DB is nullable=True?

If it was intentional, wanted to callout that it's smart and you're crushing (keep the code level non-null)

If it was an accident, want to call out to say keep doing it :)

Reply:
Was intentional for sure, seemed like an important feature